src/router.ts

```diff
@@ -1,18 +1,23 @@
 import Vue from "vue";
-import Router, { NavigationGuard } from "vue-router";
+import Router, { NavigationGuard, RawLocation } from "vue-router";
 import store from "@/store";
 import sessionStore from "@/store/modules/session";
 import Home from "./views/Home.vue";
+import Design from "./views/Design.vue";
 import Designs from "./views/Designs.vue";
 import InteractiveMap from "./views/InteractiveMap/InteractiveMap.vue";
 import Jobs from "./views/Jobs/Jobs.vue";
 import JobDetails from "./views/Jobs/JobDetails.vue";
 import Maps from "./views/Maps.vue";
 import Models from "./views/Models.vue";
 import Projects from "./views/Projects.vue";
-import NotFound from "./views/NotFound.vue";
 import PrivacyPolicy from "./views/PrivacyPolicy.vue";
 import TermsOfService from "./views/TermsOfService.vue";
+import NotFound from "./views/NotFound.vue";
+
+// FIXME (Moritz Beber): In future we would like to import this from vue-router.
```

Comment:
I remember that we discussed this previously on memote source code. Wouldn't it be better to just open an issue which references this PR (or line of code if thats possible), rather than leaving a comment in the source code?

Reply:
https://github.com/DD-DeCaF/caffeine-vue/issues/40